docker/testnet/config/signer-config.toml.sample

```diff
@@ -0,0 +1,71 @@
+[blocklist_client]
+endpoint = "http://blocklist-client:3032"
+
+[emily]
+# TODO: Add your API key here.
+endpoints = [
+    "https://YOUR_API_KEY@beta.sbtc-emily.com"
+]
+
+[bitcoin]
+# TODO: Change this to local bitcoin container
+rpc_endpoints = [
+    "http://USER:PASS@TODO:18443",
+]
+
+# TODO: Change this to local bitcoin container
+block_hash_stream_endpoints = [
+    "tcp://TODO:28332"
+]
+
+[stacks]
+# TODO: Change this to local stacks container
+endpoints = [
+    "http://TODO:20443",
+]
+
+nakamoto_start_height = 56457
+
+[signer]
+# TODO: Change this to signer private key
+private_key = "0000000000000000000000000000000000000000000000000000000000000001"
```

Comment:
nit: in the readme we say to use the `.env`, if so should we restore the `# NOTE: Set through Environment: SIGNER_SIGNER__PRIVATE_KEY`?

Reply:
Done!

Reply:
Can be done in .env as well for security, see my above comment